src/value/owned.rs

```diff
@@ -186,8 +189,14 @@ impl Struct for OwnedStruct {
 pub enum OwnedValue {
     Null(IonType),
     Integer(AnyInt),
+    Float(f64),
+    Decimal(Decimal),
+    Timestamp(Timestamp),
     String(String),
     Symbol(OwnedSymbolToken),
+    Boolean(bool),
+    Blob(Vec<u8>),
+    Clob(Vec<u8>),
```

Comment:
These **are** correct for the owned variants (just noting this in contrast to the trait API and the borrowed implementation).